pkg/traceability/sampling/sample.go

```diff
@@ -15,11 +15,26 @@ type sample struct {
 	samplingCounter    int32
 	counterResetPeriod time.Duration
 	counterResetStopCh chan struct{}
+	disableSamplingCH  chan struct{}
```

Comment:
tiny and its me just being me.....maybe match format of counterResetStopCh and update to disableSamplingCh?